Runtime/Scripts/Export/GltfWriter.cs

```diff
@@ -1076,15 +1146,36 @@ struct AttributeData {
                             vertexCount,
                             inputStreams[attrData.stream],
                             outputStreams[attrData.stream]
-                            );
+                        );
+                        break;
+                    
+                    case VertexAttribute.BlendIndices:
+                        Profiler.BeginSample("ConvertSkinningAttributesJob");
+                        var previousArraySize = outputStreams[attrData.stream].Length;
+                        // indices are uint*4 in Unity, and ushort*4 in glTF
+                        var strideDifference = (sizeof(uint) - sizeof(ushort)) * 4; 
+                        outputStreams[attrData.stream].Dispose();
+                        // gltf data will be smaller, resize the native array
+                        outputStreams[attrData.stream] = new NativeArray<byte>(previousArraySize - strideDifference * vertexCount, Allocator.TempJob);

```

Comment:
I think it should accommodate for that prior to allocating the outputStream in first place and not resize it afterwards for performance reasons.